service/src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -282,12 +284,10 @@ private ApiWorkspaceDescription buildWorkspaceDescription(
         .createdBy(workspace.createdByEmail())
         .lastUpdatedDate(
             lastChangeDetailsOptional
-                .map(ActivityLogChangeDetails::getChangeDate)
+                .map(ActivityLogChangeDetails::changeDate)
                 .orElse(OffsetDateTime.MIN))
         .lastUpdatedBy(
-            lastChangeDetailsOptional
-                .map(ActivityLogChangeDetails::getActorEmail)
-                .orElse("unknown"))
+            lastChangeDetailsOptional.map(ActivityLogChangeDetails::actorEmail).orElse("unknown"))
```

Comment:
First, I thought defaulting to unknown was handled in liquibase.

Second, can we do @Nullable/null instead of "unknown"? Usually, if an optional field isn't set, it's null. Ideally the optional fields would be @Nullable in ActivityLogChnageDetails.java.